tests/unit/src/test/scala/tests/StacktraceParseSuite.scala

```diff
@@ -0,0 +1,62 @@
+package tests
+
+import scala.meta.internal.metals.StacktraceAnalyzer
+
+class StacktraceParseSuite extends BaseSuite {
+
+  test("convert-stacktrace-line-to-symbol") {
+
+    // --- CLASS ---
+    testConversion("p1.p2.Class.method", "p1/p2/Class#")
+    testConversion(
+      "p1.p2.OutsideClass$InsideClass.method",
+      List("p1/p2/OutsideClass#", "p1/p2/OutsideClass#InsideClass#")
```

Comment:
We really just need "p1/p2/OutsideClass#", the other class must be located in the same file.

Reply:
changed